io.github.chris2511.xca.yaml

```diff
@@ -0,0 +1,68 @@
+app-id: io.github.chris2511.xca
+runtime: org.kde.Platform
+runtime-version: '5.15'
+sdk: org.kde.Sdk
+finish-args:
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=x11
+  # Qt wants OpenGL access
+  - --device=dri
+  # Wayland access
+  - --socket=wayland
+  # Needs to save files locally
+  - --filesystem=xdg-documents
+command: /app/bin/xca
+modules:
+  - name: xca
+    buildsystem: simple
+    build-commands:
+      - ./bootstrap
+      - "STRIP=: ./configure --prefix=/app"
+      - make -j$(nproc)
+    sources:
+      - type: git
+        url: https://github.com/chris2511/xca.git
+        tag: RELEASE.2.3.0
+        commit: 79510a24c52c6678e2daf3e5dbedb0841343ecea
+        disable-fsckobjects: true
+      - type: file
+        path: "io.github.chris2511.xca.appdata.xml"
```

Comment:
New appdata files should use metainfo instead. https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html

Reply:
So basically just rename xca.appdata.xml to xca.metainfo.xml ? Or are there other more subtle changes that I'm overlooking?

Reply:
Yes, just the rename.